elements/table.js

```diff
@@ -196,12 +194,15 @@ function tableElement (dats, send) {
 // ([obj], fn) -> html
 function createTable (dats, send) {
   return dats.map(dat => {
-    const progress = dat.stats.blocksTotal
-      ? Math.round(dat.stats.blocksProgress / dat.stats.blocksTotal * 100)
+    const stats = dat.stats && dat.stats.get()
```

Comment:
yeah, there's a balance with defensive programming and it's not easy to determine. however since we know there's always `dat.stats`, it's clearer this way. Otherwise you can read the code above and think otherwise.